recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "laspy" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8768499aa429c028debdff67aa738bcf323ee777025528b842f36cc22d4ccf8d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```

Comment:
Can this be noarch? (No compiled extensions or platform depended features.)

If so remove the Python 2 skip, add `noarch: python`, and add `python >=3` in `host` and `run`.

Reply:
I already did try using `python >=3` in the `host` and `run` sections locally (on Windows) and ran into the filename issue (WinError 123) also occurring in e.g. #7226. Unless that has been fixed somehow, I think we'll need to keep the skip.

Reply:
The problem is that the `>=` is part of the file name there, that is odd. However, I don't think that will be a problem for the `noarch` generated on Linux to be installed on Windows. Let's try that first and we can fix it in the feedstock later.